server/controllers/authController.js

```diff
@@ -17,7 +18,8 @@ exports.signup = async(request, response, next) => {
             email: request.body.email,
             photo: request.body.photo,
             password: request.body.password,
-            confirmPassword: request.body.confirmPassword
+            confirmPassword: request.body.confirmPassword,
+            passwordChangedAt: request.body.passwordChangedAt
```

Comment:
why do we have a `passwordChangedAt ` field?

Reply:
It will be very necessary for the future. Later on, users will need to update their passwords. When they do, I need to note the time when the password was changed. This is especially important because I would not want our users to be able to log in with a valid token, using an old password.

Reply:
Nevertheless, this is going to be an automatic input. It is not as if the user is going to be required to fill it in manually.